app/main.py

```diff
@@ -2,0 +1,12 @@
+def copy_file(command: str) -> None:
+    command = command.split()
+    if command[0] != "cp":
+        raise ValueError("You line should start with cp")
+    if len(command) != 3:
+        raise ValueError("Message should be: cp <source_file> <destination_file>")
+    if command[1] == command[2]:
```

Comment:
The error message 'There should be two different names of files' is not very clear. It would be better to say 'The source and destination file names should be different'.